src/components/Footer/index.js

```diff
@@ -10,8 +14,7 @@ export default class Footer extends Component {
   render() {
     return (
       <div className={styles.footer}>
-        <p className={styles.inline}>&lt;/&gt; with </p>
-        <span role="img" aria-label="love">❤️</span>
+        <Image src={animation} alt="animation" width="200"></Image>
```

Comment:
Does keeping it at 200 look okay on different sizes? Might wanna adjust

Reply:
I think it looks ok!
![Screen Shot 2020-07-11 at 1 46 44 PM](https://user-images.githubusercontent.com/43394060/87230301-0dea2b00-c37d-11ea-83bd-02ee3ba9cc39.png)